spring-backend/src/main/kotlin/com/example/sztoswro/member/MemberController.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.sztoswro.member
+
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class MemberController(val memberService: MemberService): MemberAPI {
+    override fun getAll(): List<MemberDTO> {
+        return memberService.getAll().map { memberDAO -> MemberDTO(memberDAO) }
+    }
+
+    override fun getById(id: Long): MemberDTO {
+        return MemberDTO(memberService.getMember(id))
+    }
+
+    override fun deleteMember(id: Long): MemberDTO {
+        TODO("Not yet implemented")
+    }
+
+    override fun addMember(memberDTO: MemberDTO) {
```

Comment:
Dodając membera raczej nie znamy jego id -> Raczej trzeba nowy model do requesta na logowanie

Reply:
ID jest wartością generowaną automatycznie w DAO. Jak będzie połączenie do bazy, to będzie brana wartość unikatowa, której nie ma w bazie i przypisywana do membera. Wtedy powinniśmy po unikatowych usernamach (adresach email) sprawdzać czy nie istnieje taki użytkownik w bazie.